e2e/workspaces/demo_app/open_link.yaml

```diff
@@ -0,0 +1,32 @@
+appId: com.example.example
+tags:
+    - passing
+---
+- launchApp:
+    appId: com.android.chrome
+    clearState: true
+    optional: true
+
+- launchApp:
+    clearState: true
+
+- openLink: 'https://maestro.mobile.dev'
+- assertVisible:
+    id: 'signin_fre_dismiss_button|send_report_checkbox|terms_accept'
```

Comment:
What's this asserting? I'm unclear why this isn't the docs. 
On that point, is it worth a comment for future people maintaining this test?